### PR TITLE
Orchestration reportingevent

### PR DESCRIPTION
**Pull request by anshuman-yadav**
Please link to the issue #…
Added an addition field in reportingevent Schema by name flowControlID

### CODE REVIEW

File: extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json [hunk 1]
```diff
@@ -51,6 +51,13 @@
             "condition": "if a conditional step has been processed"
           }
         },
+        "xdm:flowControlID": {
+          "title": "Unique identifier for the flow control",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The identifier of the flow control in a journey verison"
```
Comment (lrosenthol) [inline extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json]: What is a "journey", seems very product specific.  Also, I assume "verison" is "version", please fix.

File: extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json [hunk 2]
```diff
@@ -51,6 +51,13 @@
             "condition": "if a conditional step has been processed"
           }
         },
+        "xdm:flowControlID": {
```
Comment (lrosenthol) [inline extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json]: flow control seems like a product-specific thing, is there a generic term?

File: extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json (hunk 1 above)
Comment (anshuman-yadav) [reply extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json]: Journey is product specific in our case ( explained in the description file of this same schema ) , I have fixed the version typo

File: extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json (hunk 2 above)
Comment (anshuman-yadav) [reply extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json]: I will add in the description what a flow control in the Journey means ( the product specific definition ). 
I will comment on the PR after the changes are committed and can be reviewed.

**Comment by anshuman-yadav** [comment]
@lrosenthol ... you can review the changes now.

File: extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json (hunk 2 above)
Comment (lrosenthol) [reply extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json]: and this is an extension, so product-specific stuff is permitted.  OK.

**Pull request merged**